src/lib/utils/string.ts

```diff
@@ -39,6 +39,8 @@ function convertToKebabSnakeCase(str: string, caseType: keyof typeof cases) {
 }
 
 export function convertCase(str: string, fromCase: keyof typeof cases, toCase: keyof typeof cases) {
+  const uncap = uncapitalize(str);
+
```

Comment:
Don't put it here, it should be inside the "pascal" arm case as it shouldn't run for all case arms

Reply:
It's because of ESLint, the function needs to be refactored anyway.